pkg/policyfilter/k8s_test.go

```diff
@@ -302,10 +302,46 @@ func (ts *testState) podsCgroupIDs(t *testing.T, podNames ...string) []uint64 {
 	return ret
 }
 
+// get cgroup IDs of containers from pods
+// podContainerMap is a map of podName -> [podContainerNames]
+func (ts *testState) containersCgroupIDs(t *testing.T, podContainerMap map[string][]string) []uint64 {
+	var ret []uint64
+	for podName, containerNames := range podContainerMap {
+		var p *testPod
+		for _, pod := range ts.pods {
+			if pod.name == podName {
+				p = &pod
+				break
+			}
+		}
+
+		if p == nil {
+			t.Fatalf("unknown pod name: %s", podName)
```

Comment:
Let's not have `Fatal` here, unless we really want to crash the agent which I don't think we do.

Reply:
Ah you are right!
I missed that it was a `t.` `fatal` in my hasty review :)